hw/ip/otp_ctrl/doc/checklist.md

```diff
@@ -10,21 +10,21 @@ stage, and updated as needed. Once done, please remove this comment before check
 This checklist is for [Hardware Stage]({{< relref "/doc/project/development_stages.md" >}}) transitions for the [OTP_CTRL peripheral.]({{< relref "hw/ip/otp_ctrl/doc" >}})
 All checklist items refer to the content in the [Checklist.]({{< relref "/doc/project/checklist.md" >}})
 
-## Design Checklist
+<%text>## Design Checklist</%text>
 
-### D1
+<%text>### D1</%text>
 
 Type          | Item                  | Resolution  | Note/Collaterals
 --------------|-----------------------|-------------|------------------
-Documentation | [SPEC_COMPLETE][]     | Not Started | [OTP_CTRL Design Spec]({{<relref "hw/ip/otp_ctrl/doc" >}})
-Documentation | [CSR_DEFINED][]       | Not Started |
-RTL           | [CLKRST_CONNECTED][]  | Not Started |
-RTL           | [IP_TOP][]            | Not Started |
-RTL           | [IP_INSTANTIABLE][]   | Not Started |
-RTL           | [MEM_INSTANCED_80][]  | Not Started |
-RTL           | [FUNC_IMPLEMENTED][]  | Not Started |
-RTL           | [ASSERT_KNOWN_ADDED][]| Not Started |
-Code Quality  | [LINT_SETUP][]        | Not Started |
+Documentation | [SPEC_COMPLETE][]     | Done        | [OTP_CTRL Design Spec]({{<relref "hw/ip/otp_ctrl/doc" >}})
+Documentation | [CSR_DEFINED][]       | Done        |
+RTL           | [CLKRST_CONNECTED][]  | Done        |
+RTL           | [IP_TOP][]            | Done        |
+RTL           | [IP_INSTANTIABLE][]   | Done        |
+RTL           | [MEM_INSTANCED_80][]  | Done        |
+RTL           | [FUNC_IMPLEMENTED][]  | Done        |
+RTL           | [ASSERT_KNOWN_ADDED][]| Done        |
+Code Quality  | [LINT_SETUP][]        | Done        |
 Review        | Reviewer(s)           | Not Started |
```

Comment:
Do we need a reviewer for D1?

Reply:
@sjgitty I thought this is not needed anymore, right? should we delete these last two lines from the template?

Reply:
> @sjgitty I thought this is not needed anymore, right? should we delete these last two lines from the template?

Yes, I think it should be removed from the template. Feel free to remove now, or if you keep it we'll remove from all checklists when we do the template PR.

Reply:
ok let's do that as a separate cleanup in order to not "pollute" the D0 -> D1 transition PR of OTP.